<launch>
  <!-- Arguments -->
  <arg name="camera_file"       default="$(find unstruct_navigation_ros)/config/resized_wide_angle_dual.yaml"/>
  <arg name="params_file"       default="$(find unstruct_navigation_ros)/config/default.yaml"/>  

  <!-- Clear old params -->  
  <rosparam command="delete" param="unstruct_planner_node"/>

  <!-- Load parameters -->
  <rosparam command="load" file="$(arg params_file)" ns="unstruct_planner_node"/>
  <rosparam command="load" file="$(arg camera_file)" ns="unstruct_planner_node"/>

  <node name="unstruct_planner_node" pkg="unstruct_navigation_ros" type="unstruct_planner_node.py" output="screen">
    <param name="reload_default_params" value="False" />
  </node>

</launch>
